account.go

```diff
@@ -274,7 +280,7 @@ type AccountCapabilitiesParams struct {
 	CartesBancairesPayments *AccountCapabilitiesCartesBancairesPaymentsParams `form:"cartes_bancaires_payments"`
 	EPSPayments             *AccountCapabilitiesEPSPaymentsParams             `form:"eps_payments"`
 	FPXPayments             *AccountCapabilitiesFPXPaymentsParams             `form:"fpx_payments"`
-	GiropayPayments         *AccountCapabilitiesGiropayPaymentsParams         `form:"giropay_payments"`
+	GrabpayPayments         *AccountCapabilitiesGrabpayPaymentsParams         `form:"grabpay_payments"`
```

Comment:
This mistakenly removes Giropay I think.
```suggestion
	GiropayPayments         *AccountCapabilitiesGiropayPaymentsParams         `form:"giropay_payments"`
	GrabpayPayments         *AccountCapabilitiesGrabpayPaymentsParams         `form:"grabpay_payments"`
```

Reply:
Oops, nice catch, PTAL